adapters/IronSource/IronSourceAdapter/GADMAdapterIronSource.h

```diff
@@ -16,6 +16,6 @@
 #import <GoogleMobileAds/GoogleMobileAds.h>
 
 /// Adapter for communicating with the IronSource Network to fetch nterstitial video ads.
```

Comment:
`banner and interstitial ads`? Does ironSource only do video for interstitial?

Reply:
Done.